src/router.ts

```diff
@@ -77,7 +77,9 @@ class Router {
     await fetchMMOrder(symbol).then(async (order) => {
       // Run the risk manager if there is no MM order
       // @ts-ignore
-      if (!order || Number(order.remainingQuantity) < dipDeposit.qtyTokens) {
+      if (!order || order.price === undefined
```

Comment:
For my knowledge,  doesn't !order already catch undefined? This is the most common failure from the API I have observed when the MM bid isn't active

Reply:
I just changed the API to return a JSON string with an error message. So there is a response, it just doesnt have a price field

Reply:
This isnt really a failure from the API. There just isnt an MM bid for a lot of the time. For example, we cannot offload the 2 sol for tomorrow